.github/workflows/ci.yml

```diff
@@ -45,6 +45,9 @@ jobs:
         run: |
           npm run lint
           npm run test:unit
+      - name: test Demo
+        run: |
+          cd demo && npm i && npm run build
```

Comment:
Vite being able to build the app tells us very little (beyond that we have correct file paths, as you've spotted & corrected). Whether these modules actually work or not wouldn't be caught at all.

This is more of a test of Vite than Preact, I don't think it makes sense to be part of the CI as-is. If we did want to do an ecosystem CI (and that might be a very good idea), it would need proper tests and what have you. `/demo` just isn't that at the moment. I don't see the value-add in testing it.

Reply:
I agree, let's not add arbitrary things to our CI that don't involve anything Preact related. This demo tests more that vite bundles this than anything else, so it would be better fit for preset-vite